src/components/Toast/GalleryToast.tsx

```diff
@@ -0,0 +1,182 @@
+import { FaceSmileIcon, HandThumbUpIcon } from '@heroicons/react/24/solid';
+import Button from '../Button/Button';
+import { toast } from './useToast';
+import { CloudArrowUpIcon } from '@heroicons/react/24/outline';
+import { IconEmojiTalkingAngry } from '@allbin/icons';
+
+const GalleryToast = () => {
+  return (
+    <div className="grid grid-cols-2 gap-2">
+      <Button onClick={() => toast({ title: 'Toast!!' })}>Normal toast</Button>
+      <Button onClick={() => toast.success({ title: 'Toast success!!' })}>
+        Success toast
+      </Button>
+      <Button
+        onClick={() =>
+          toast.success(
+            { title: 'Success 999999ms autoClose!' },
+            { autoClose: 999999 },
+          )
+        }
+      >
+        Success toast (autoClose 999999ms)
+      </Button>
+      <Button onClick={() => toast.info({ title: 'Toast info!!' })}>
+        Info toast
+      </Button>
+      <Button
+        onClick={() =>
+          toast.info({
+            title: 'Toast info!!',
+            icon: <FaceSmileIcon className="size-12" />,
+          })
+        }
+      >
+        Info toast with BIG custom icon
+      </Button>
+      <Button
+        onClick={() =>
+          toast.info({
+            title:
+              "Toast info with a very long text that is just very long and doesn't stop. Just continues for way to loooooong!!!",
+          })
+        }
+      >
+        Very long info toast
+      </Button>
+      <Button onClick={() => toast.warning({ title: 'Toast warning!!' })}>
+        Warning toast
+      </Button>
+      <Button
+        onClick={() =>
+          toast.warning(
+            { title: 'Toast warning!! (Click me anywhere to close me)' },
+            { closeButton: false, closeOnClick: true, autoClose: 10000 },
+          )
+        }
+      >
+        Warning toast (no close button, close on click anywhere in toast)
+      </Button>
+      <Button
+        onClick={() =>
+          toast.error(
+            { title: 'Toast error with long autoClose time!' },
+            { autoClose: 999999 },
+          )
+        }
+      >
+        Error toast (autoClose time 999999ms)
+      </Button>
+      <Button
+        onClick={() =>
+          toast.error({ title: 'Toast error!!' }, { autoClose: 500 })
+        }
+      >
+        Error toast (autoClose time 500ms)
```

Comment:
Feels wrong to say that the close time is 500ms but in reality error toasts never close automatically